File: Employee.java
import java.util.ArrayList;

public class Employee {
    private String name;
    private String lastName;
    private ArrayList<BankAccount> accountList = new ArrayList<>();
    //Método Constructor de la clase
    public Employee(String name, String lastName) {
        this.name = name;
        this.lastName = lastName;
    }
    //Getters y setters de la clase
    public String getName() {
        return name;
    }
    public void setName(String name) {
        this.name = name;
    }
    public String getLastName() {
        return lastName;
    }
    public void setLastName(String lastName) {
        this.lastName = lastName;
    }
    public ArrayList<BankAccount> getAccountList() {
        return accountList;
    }
    //Métodos para añadir o eliminar cuentas del atributo ArrayList del empleado
    public void addAccount(BankAccount account){
        accountList.add(account);
    }
    public void deleteAccount(BankAccount account){
        accountList.remove(account);
    }
}

File: Employee.py
class Employee:
    def __init__(self, name, last_name):
        self.name = name
        self.last_name = last_name
        self.account_list = []

    def get_name(self):
        return self.name

    def set_name(self, name):
        self.name = name

    def get_last_name(self):
        return self.last_name

    def set_last_name(self, last_name):
        self.last_name = last_name

    def get_account_list(self):
        return self.account_list

    def add_account(self, account):
        self.account_list.append(account)

    def delete_account(self, account):
        self.account_list.remove(account)